lib/android/app/src/main/java/com/reactnativenavigation/parse/AnimationOptions.java

```diff
@@ -0,0 +1,68 @@
+package com.reactnativenavigation.parse;
+
+
+import org.json.JSONObject;
+
+public class AnimationOptions {
+
+    public static AnimationOptions parse(JSONObject json) {
+        AnimationOptions options = new AnimationOptions();
+        if (json == null) return options;
+
+        options.empty = false;
+
+        options.y = ValueAnimationOptions.parse(json.optJSONObject("y"));
+        options.x = ValueAnimationOptions.parse(json.optJSONObject("x"));
+        options.rotationX = ValueAnimationOptions.parse(json.optJSONObject("rotationX"));
+        options.rotationY = ValueAnimationOptions.parse(json.optJSONObject("rotationY"));
+        options.rotation = ValueAnimationOptions.parse(json.optJSONObject("rotation"));
+        options.alpha = ValueAnimationOptions.parse(json.optJSONObject("alpha"));
+        options.scaleX = ValueAnimationOptions.parse(json.optJSONObject("scaleX"));
+        options.scaleY = ValueAnimationOptions.parse(json.optJSONObject("scaleY"));
+
+        return options;
+    }
+
+    private boolean empty = true;
```

Comment:
Can we use positive (hasValue) instead of negative (isEmpty)? This help avoid double negation conditional which are very confusing.